swig/comedi.i

```diff
@@ -115,14 +118,19 @@ unsigned int NI_AO_SCAN_BEGIN_SRC_RTSI(unsigned int rtsi_channel);
 %array_class(lsampl_t, lsampl_array);
 %array_class(comedi_insn, insn_array);
 
+#ifndef SWIGPYTHONONLYSHORT
 %insert("python") %{
 # Add entries in module dictionary to strip comedi_/COMEDI_ prefix
+import copy
 import re
-for k,v in globals().items():
+myglobals = copy.copy(globals())
```

Comment:
You could make it just a little simpler by writing:
`for k,v in globals().copy().items():`

(And in turn this would avoid importing copy.copy and creating a new variable)